airflow-ctl/src/airflowctl/api/operations.py

```diff
@@ -147,6 +150,42 @@ def __init_subclass__(cls, **kwargs):
             if callable(value):
                 setattr(cls, attr, _check_flag_and_exit_if_server_response_error(value))
 
+    def execute_list(
+        self,
+        *,
+        path: str,
+        data_model: type[T],
+        offset: int = 0,
+        limit: int = 50,
+        params: dict | None = None,
+        **kwargs,
+    ) -> T | ServerResponseError:
+        shared_params = {**(params or {}), **kwargs}
+        try:
+            self.response = self.client.get(path, params=shared_params)
+            first_pass = data_model.model_validate_json(self.response.content)
+            total_entries = first_pass.total_entries  # type: ignore[attr-defined]
```

Comment:
This is done two times, we don't need that.

total_entries = first_pass.total_entries # type: ignore[attr-defined]